ci/environment.yml

```diff
@@ -13,3 +13,6 @@ dependencies:
   - pytest-cov
   - scipy
   - xarray
+  # Tools run by pre-commit so they can be in the same conda environment
```

Comment:
Conda environment is supposed to represent the minimum required dependencies for development purposes. These tools are configured and run by pre-commit through the [.pre-commit-config.yaml](https://github.com/UXARRAY/uxarray/blob/main/.pre-commit-config.yaml) file and do not need to be in the conda environment.